Gopkg.lock

```diff
@@ -1370,6 +1370,7 @@
     "go.uber.org/zap/zaptest",
     "go.uber.org/zap/zaptest/observer",
     "golang.org/x/xerrors",
+    "k8s.io/api/apps/v1",
```

Comment:
Hmm...I don't see this new import being used anywhere in this commit. Is this needed?

Reply:
humm, not sure, this is autogenerated ...

Reply:
this disappeared after rebasing with master 🤔